Tests/BackendIntegrationTests/LoadShedderIntegrationTests.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  LoadShedderIntegrationTests.swift
+//
+//  Created by Nacho Soto on 3/21/23.
+
+import Nimble
+@testable import RevenueCat
+import StoreKit
+import XCTest
+
+class LoadShedderStoreKit2IntegrationTests: LoadShedderStoreKit1IntegrationTests {
+
+    override class var storeKit2Setting: StoreKit2Setting {
+        return .enabledForCompatibleDevices
+    }
+
+}
+
+class LoadShedderStoreKit1IntegrationTests: BaseStoreKitIntegrationTests {
+
+    override var apiKey: String { return Constants.loadShedderApiKey }
+
+    override class var storeKit2Setting: StoreKit2Setting {
+        return .disabled
+    }
+
+    func testCanGetOfferings() async throws {
+        let receivedOfferings = try await Purchases.shared.offerings()
+        expect(receivedOfferings.all).toNot(beEmpty())
+    }
+
+    func testCanPurchasePackage() async throws {
+        try await self.purchaseMonthlyOffering()
+    }
```

Comment:
I guess we will need to add one as well for getting the `ProductEntitlementMapping` as well?

Reply:
I did that in #2355 for the normal server. Once that's done I'll add it here too 👍🏻

Reply:
Done here now.